tests/schemas/api_v3/transactionStatistics.schema.js

```diff
@@ -40,10 +40,10 @@ const transactionStatisticsSchema = {
 	distributionByAmount: Joi.object().required(),
 };
 
-const date = {
-	dateFormat: Joi.string().valid(...allowedDateFormats).required(),
-	dateFrom: Joi.string().required(),
-	dateTo: Joi.string().required(),
+const duration = {
+	format: Joi.string().valid(...allowedDateFormats).required(),
+	from: Joi.string().required(),
+	to: Joi.string().required(),
```

Comment:
Please add a new regex: `DURATION: /^\d{4}-\d{2}(?:-\d{2})?$/`

```suggestion
	from: Joi.string().pattern(regex.DURATION).required(),
	to: Joi.string().pattern(regex.DURATION).required(),
```

Reply:
applied.